tests/get-model.cpp

```diff
@@ -11,7 +11,7 @@ char * get_model_or_exit(int argc, char *argv[]) {
 
     } else {
         model_path = getenv("LLAMACPP_TEST_MODELFILE");
-        if (!model_path || strlen(model_path) == 0) {
+        if (!model_path || model_path[0] == '\0') {
```

Comment:
In this case where performance is not a concern, I would prefer the clarity of `strlen`.

Reply:
I would personally default to writing this as `!model_path || !*model_path`, but maybe that's too terse.

Reply:
Yes, you're right. For single entry at the tests, I think it's okay to sacrifice a bit of performance for the sake of readability.